tests/dbt/test_graph.py

```diff
@@ -108,6 +108,29 @@ def test_load(
     assert load_function.called
 
 
+@pytest.mark.integration
+@patch("cosmos.dbt.graph.Popen")
+def test_load_via_dbt_ls_uses_temp_dir(mock_popen):
+    mock_popen().communicate.return_value = ("", "")
+    dbt_project = DbtProject(name="jaffle_shop", root_dir=DBT_PROJECTS_ROOT_DIR)
+    dbt_graph = DbtGraph(
+        project=dbt_project,
+        profile_config=ProfileConfig(
+            profile_name="default",
+            target_name="default",
+            profile_mapping=PostgresUserPasswordProfileMapping(
+                conn_id="airflow_db",
+                profile_args={"schema": "public"},
+            ),
+        ),
+    )
+
+    dbt_graph.load_via_dbt_ls()
+    used_cwd = Path(mock_popen.call_args[0][0][-5])
+    assert used_cwd != dbt_project.dir
+    assert not used_cwd.exists()
```

Comment:
this doesn't look like it's confirming that files are written to the temp directory, right?

Reply:
Just that they are not being written to the project dir and that the directory they were written to no longer exists.
Temp directories naming can vary depending on the OS, so I thought this was safer. Do you recommend any alternative?